maint/ManyConfigTests

```diff
@@ -311,8 +309,8 @@ if [ $ISGCC -ne 0 -a $usemain -ne 0 ]; then
   fi
 # This also seems to be the case for sanitize undefined.
   if [ $useusan -ne 0 ]; then
-    echo "------- Maximally configured test with -fsanitize=undefined -fno-sanitize=shift -fno-sanitize=alignment -std=gnu99 -------"
-    CFLAGS="$OFLAGS $SAVECFLAGS -no-pie -fno-PIE -fsanitize=undefined -fno-sanitize=shift -fno-sanitize=alignment -std=gnu99"
+    echo "------- Maximally configured test with -fsanitize=undefined -fno-sanitize=alignment -std=gnu99 -------"
```

Comment:
I couldn't find any explanation on why `-fno-sanitize=alignment` was originally added, but I don't think it might be needed anymore (sljit had several UB fixes in the last years).